### PR TITLE
Truncate redundant job strings and set a real commit_count.

### PR DESCRIPTION
**Pull request by david-mcmahon**
And further, remove "jenkins" references.

### CODE REVIEW

File: lib/releaselib.sh [hunk 1]
```diff
@@ -219,15 +223,10 @@ release::set_build_version () {
...
-        #       So we can return a message about how far ahead the top of the
-        #       release branch is from the last good commit.
-        #commit_count=$(git rev-list $build_sha1..${branch_head} |wc -l)
-        commit_count=some
+        commit_count=$((first_build_number-build_number))
         logecho
         logecho "$ATTENTION: The $branch branch HEAD is ahead of the last" \
-                "good Jenkins run by $commit_count commits." \
-                "Wait for Jenkins to catch up."
+                "good run by $commit_count commits."
```
Comment (enisoc) [inline lib/releaselib.sh]: Probably more accurate to say "by $build_count builds"? We don't necessarily create a build for every commit.

Comment (david-mcmahon) [reply lib/releaselib.sh]: Understood that the CI can skip commits, but this is counting the actual number of commits.

Comment (enisoc) [reply lib/releaselib.sh]: Oh how does that work?

Comment (enisoc) [reply lib/releaselib.sh]: Ohh I see. It's coming from the tag calculated by git. Clever! :)

Comment (david-mcmahon) [reply lib/releaselib.sh]: Yeah I realized it by accident when looking at something else.  I thought that TODO would be there forever. :)

**Pull request merged**